watch/src/blockprint/database.rs

```diff
@@ -33,6 +33,7 @@ pub struct WatchBlockprint {
 }
 
 #[derive(Debug, QueryableByName, diesel::FromSqlRow)]
+#[allow(dead_code)]
 pub struct WatchValidatorBlockprint {
```

Comment:
should this be deleted? i don't see it used but not sure if it's a useful export if anything depends on beacon watch?